float64 value
bool error